DatadogSessionReplay/Tests/Recorder/Utilties/ImageDataProviderTests.swift

```diff
@@ -27,7 +27,7 @@ class ImageDataProviderTests: XCTestCase {
     @available(iOS 13.0, *)
     func test_returnsValidString_forSFSymbolIcon() throws {
         let sut = ImageDataProvider()
-        let image = UIImage(systemName: "apple.logo")
+        let image = UIImage(systemName: "square.and.arrow.up")
```

Comment:
The `apple.logo` symbols is only available on iOS 16+. Replacing it with one that is there since iOS 13.0.

<img width="200" alt="Screenshot 2023-07-28 at 09 19 50" src="https://github.com/DataDog/dd-sdk-ios/assets/2358722/dc6b65fd-193c-4e33-8f3e-104583ccbcaa">